redis/src/cluster_client.rs

```diff
@@ -306,6 +313,26 @@ impl ClusterClientBuilder {
         self
     }
 
+    /// Configure hostname verification when connecting with TLS.
+    ///
+    /// If `insecure` is true, this **disables** hostname verification, while
+    /// leaving other aspects of certificate checking enabled (but that isn't
```

Comment:
"(but that isn't worth much: without hostname verification, TLS as a whole is not secure)" -> I'd prefer a phrasing that sounds more... "official", maybe?
"... checking enabled, this isn't secure enough for production-grade applications." maybe?

Reply:
I just deleted the part in parentheses. The warning that follows is clear and not too casual.